pkg/ciliumenvoyconfig/cell.go

```diff
@@ -31,8 +33,17 @@ var Cell = cell.Module(
 	cell.ProvidePrivate(newCECManager),
 	cell.ProvidePrivate(newCECResourceParser),
 	cell.ProvidePrivate(newEnvoyServiceBackendSyncer),
+	cell.Config(cecConfig{}),
 )
 
+type cecConfig struct {
+	EnvoyConfigRetryFailedInterval time.Duration
+}
+
+func (r cecConfig) Flags(flags *pflag.FlagSet) {
+	flags.Duration("envoy-config-retry-failed-interval", 15*time.Second, "Interval in which an attempt is made to reconcile failed EnvoyConfigs. If the duration is zero, the retry is deactivated.")
```

Comment:
IMO `envoy-config-retry-interval` would be more succinct. "retry" kind of implies it is done on failure?

Reply:
agree, changed! thanks!